helpdesk_mgmt/views/helpdesk_ticket_templates.xml

```diff
@@ -11,143 +11,187 @@
                         </div>
-                        <div class="col-md-6">
-                          <strong>Last Stage Update:</strong> <span t-field="ticket.last_stage_update" /><br/>
-                          <strong>Attachments:</strong><br />
-                          <t t-foreach="files" t-as="f">
-                              <a t-att-href="'/web/content/%i?download=true' % f.id">
-                                  <span class="fa fa-download" />
-                                  <span t-esc="f.name" />
-                              </a>
-                              <br />
-                          </t>
+                        <div class="col-12 col-md-6" t-if="ticket.closed_date">
+                            <strong>Closed Date:</strong> <span t-field="ticket.closed_date" t-options='{"widget": "date"}'/>
                         </div>
-                      </div>
-                      <div class="row mt8">
-                        <div class="col-md-12">
-                          <h4 class="page-header">Description</h4>
-                          <t t-raw="ticket.description"/><br/>
+                        <div class="col-12 col-md-6 mb-1">
+                            <strong>Last Stage Update::</strong> <span t-field="ticket.last_stage_update" t-options='{"widget": "date"}'/>
                         </div>
-                      </div>
                     </div>
-                    <h4 class="page-header">History</h4>
-                    <!-- Options:Ticket Chatter: user can reply -->
-                    <t t-call="portal.message_thread">
-                      <t t-set="object" t-value="ticket"/>
-                    </t>
-
-                  </div>
-                </div>
-              </div>
-          <div class="oe_structure mb32"/>
-      </t>
-  </template>
+                    <div class="row mb-4" t-if="ticket.user_id or ticket.partner_id">
+                        <div class="coll-12 col-md-6 pb-2" t-if="ticket.partner_id">
+                            <strong>Reported by</strong>
+                            <div class="row">
+                                <div class="col flex-grow-0 pr-3">
+                                    <img t-if="ticket.partner_id.image" class="rounded-circle mt-1 o_portal_contact_img" t-att-src="image_data_uri(ticket.partner_id.image)" alt="Contact"/>
+                                    <img t-else="" class="rounded-circle mt-1 o_portal_contact_img" src="/web/static/src/img/user_menu_avatar.png" alt="Contact"/>
+                                </div>
+                                <div class="col pl-md-0">
+                                    <div t-field="ticket.partner_id" t-options='{"widget": "contact", "fields": ["name", "email", "phone"]}'/>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 col-md-6 pb-2" t-if="ticket.user_id">
```

Comment:
IMHO, the assigned user is just a bonus (optional). Anyway, it can be added or removed  by XPATH.